Api/src/controllers/userController.js

```diff
@@ -0,0 +1,104 @@
+// import {
+//   v4 as uuidv4
+// } from 'uuid';
+
+import data from "../data.json" assert { type: "json" };
+
+// let users = data.Users;
+let users = [];
+
+export const getUsers = (req, res) => {
+  res.send({
+    status: 200,
+    users: data.Users
+  });
+};
+
+export const createUser = (req, res) => {
+  const {
+    id,
+    firstName,
+    lastName,
+    age,
+    email,
+    phoneNumber,
+    passportURL,
+  } = req.body;
+
+  const user = {
```

Comment:
Usually, the shorthand is prefered. 
```suggestion
  const user = {
    id,
    firstName,
    lastName,
    age,
    email,
    phoneNumber,
    passportURL,
    isAdmin: false,
  }
```

It seems like your codebase doesn't have `eslint` configured or else these would be flagged to you.

Reply:
Yeah, I've fixed this thanks.